cluster/gce/gci/configure-helper.sh

```diff
@@ -652,8 +652,13 @@ function create-master-auth {
     append_or_replace_prefixed_line "${known_tokens_csv}" "${GCE_GLBC_TOKEN},"                "system:controller:glbc,uid:system:controller:glbc"
   fi
   if [[ -n "${ADDON_MANAGER_TOKEN:-}" ]]; then
-    append_or_replace_prefixed_line "${known_tokens_csv}" "${ADDON_MANAGER_TOKEN},"   "system:addon-manager,uid:system:addon-manager,system:masters"
+    append_or_replace_prefixed_line "${known_tokens_csv}" "${ADDON_MANAGER_TOKEN},"           "system:addon-manager,uid:system:addon-manager,system:masters"
```

Comment:
s/           / /

Reply:
(there should be 1 space instead of a bunch of spaces and a tab there)

Reply:
actually just revert this change?

Reply:
This fixes the alignment with the previous lines. Would you still like me to revert?

Reply:
ack, sorry, the diff obscured this.